assignments/array-methods.js

```diff
@@ -56,28 +56,45 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+runners.forEach((cur) => {
+    fullName.push(`${cur["first_name"]} ${cur["last_name"]}`);
+});
```

Comment:
You could also use [object destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring) to get ahold of the values in this way:
```javascript
runners.forEach(({ first_name, last_name }) => {
    fullName.push(`${first_name} ${last_name}`);
});
```